requirements-dev.txt

```diff
@@ -1,14 +1,13 @@
 black~=22.0
 stestr
-Sphinx>=3.0.0, <=6.2.1
+Sphinx>=5.0.0, <=6.2.1
 pylint==2.7.1
 reno>=3.2.0
-qiskit-sphinx-theme>=1.6, <=1.10.3
+qiskit-sphinx-theme~=1.11.1
```

Comment:
This pins to 1.11 so that your docs don't break overnight on you. We are trying to keep the theme stable, but it is undergoing rapid improvements currently, so it's safer to pin.

I will open PRs to update to future releases.